issuesync.rb

```diff
@@ -160,7 +160,7 @@ def get_env key
 
     def http
       @http ||= begin
-        conn = Net::HTTP::Persistent.new self.class.name, proxy_uri
+        conn = Net::HTTP::Persistent.new name: self.class.name, proxy: proxy_uri
```

Comment:
When did Net::HTTP::Persistent start accepting named attributes? Will older Net::HTTP::Persistent remain compatible with this? Can we ensure compatibility with both old and new Net::HTTP::Persistent?

Reply:
> When did Net::HTTP::Persistent start accepting named attributes?

It happened between Net::HTTP::Persistent [2.x and 3.0](https://github.com/drbrain/net-http-persistent/pull/84).

> Can we ensure compatibility with both old and new Net::HTTP::Persistent?

Seems like this oneliner can get the installed version of a module.

`ruby -e 'puts Gem::Specification.find_all_by_name("net-http-persistent").map(&:version)'`

So maybe, a conditional block that would support both old and new conventions?

Reply:
To avoid the version check, I've tried if checking constructor arity would work, but for some reason it doesn't:

```rb
# v2.9.4
Net::HTTP::Persistent.method(:new).arity #=> -1
Net::HTTP::Persistent.method(:new).parameters #=> [[:rest:]]

# v3.0.0
Net::HTTP::Persistent.method(:new).arity #=> -1
Net::HTTP::Persistent.method(:new).parameters #=> [[:rest:]]
```

There seems to be no difference. So instead let's do the version check:

```rb
if Net::HTTP::Persistent::VERSION.to_i < 3
  Net::HTTP::Persistent.new self.class.name, proxy_uri
else
  Net::HTTP::Persistent.new name: self.class.name, proxy: proxy_uri
end
```

Reply:
The version check LGTM :+1: